src/styles/global.css

```diff
@@ -338,4 +338,8 @@ footer a {
     font-size: 50px;
     font-weight: bolder;
     line-height: 90px;
+}
+
+.about p, .about ul {
+    font-size: 1.5em;
```

Comment:
Why increase the font-size this dramatically? At least on my screen this looks way too large, and removing this styling looked better to me.

![Screenshot from 2022-06-04 15-35-40](https://user-images.githubusercontent.com/3187531/172024776-1426d8bd-99ba-4d82-98f3-22852f6fbf1a.png)